src/doublyLinkedList.js

```diff
@@ -0,0 +1,212 @@
+'use strict'
+
+function Node(value){
+  this._data = value
+  this._next = null
+  this._prev = null
+
+  this.data = () => this._data
+  this.next = () => this._next
+  this.prev = () => this._prev
+}
+
```

Comment:
What are the purpose of these functions? shouldn't we just have the underscored properties and set the value directly to those?